### ISSUE TITLE
How to generate training set and test set with fixed SNR

### ISSUE THREAD

**Issue by wozaimoyu** [lxf8519/DL-hybrid-precoder#6]
I preliminary contact deep learning, want to know how to generate fixed signal-to-noise ratio training set and test set, paper “DeepMIMO: A Generic Deep Learning Dataset for Millimeter Wave and Massive MIMO Applications” I do not quite understand
